common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.group_type.normal.description": "Players who are not in your group can't hear you, but you can hear them.",
   "message.voicechat.group_type.open": "Open",
-  "message.voicechat.group_type.open.description": "Players that are not in your group can hear you and you can hear them",
+  "message.voicechat.group_type.open.description": "Players who are not in your group can hear you, and you can hear them too.",
   "message.voicechat.group_type.isolated": "Isolated",
-  "message.voicechat.group_type.isolated.description": "Players that are not in your group can't hear you and you can't hear them",
-  "message.voicechat.join_group": "Join group",
+  "message.voicechat.group_type.isolated.description": "Players who are not in your group can't hear you, and you can't hear them either.",
+  "message.voicechat.join_group": "Join Group",
   "message.voicechat.enter_group_password": "Enter Group Password",
   "message.voicechat.password": "Password",
-  "message.voicechat.wrong_password": "Wrong Password",
+  "message.voicechat.wrong_password": "Incorrect Password",
   "message.voicechat.server_port": "Voice chat server hosted on port %s",
-  "message.voicechat.macos_no_mic_permission": "No microphone permission. Please check Security & Privacy settings",
-  "message.voicechat.macos_unsupported_launcher": "Your launcher does not support MacOS microphone permissions",
+  "message.voicechat.macos_no_mic_permission": "No permission to access your microphone. Please open System Settings > Privacy & Security and check the access options there",
+  "message.voicechat.macos_unsupported_launcher": "Your launcher does not support macOS microphone permissions",
   "message.voicechat.player_no_voicechat": "%s does not have %s installed",
-  "message.voicechat.mute.enabled": "Press to unmute your microphone",
-  "message.voicechat.mute.disabled": "Press to mute your microphone",
-  "message.voicechat.mute.disabled_ptt": "You can't mute when using push to talk",
-  "message.voicechat.disable.enabled": "Press to enable all voice chat sounds",
-  "message.voicechat.disable.disabled": "Press to disable all voice chat sounds",
-  "message.voicechat.disable.no_speaker": "No speaker available",
-  "message.voicechat.hide_icons.enabled": "Press to show all voice chat icons",
-  "message.voicechat.hide_icons.disabled": "Press to hide all voice chat icons",
-  "message.voicechat.show_group_hud.enabled": "Press to hide the group chat HUD",
-  "message.voicechat.show_group_hud.disabled": "Press to show the group chat HUD",
-  "message.voicechat.recording.disabled": "Press to start recording",
-  "message.voicechat.recording.enabled": "Press to stop recording",
+  "message.voicechat.mute.enabled": "Press to unmute your microphone.",
+  "message.voicechat.mute.disabled": "Press to mute your microphone.",
+  "message.voicechat.mute.disabled_ptt": "You can't mute your microphone when using Push to Talk.",
```

Comment:
I will answer this question later, after other conversations are concluded.

Reply:
Now that we've agreed on the capitalization style for button labels, I can get back to answering the question that you asked here a while ago.

I made this change because of the following screen:
![Activation-method-selection-screen](https://github.com/user-attachments/assets/2916d7c1-d51b-4c4b-ab4e-f1c037d63942)

In this screen, terms like "Push to Talk" and "Voice Activation" appear in the middle of a sentence and are capitalized (which is a side effect of reusing the same strings as the button labels at the bottom, which is a very clever solution and I like it a lot, as it keeps the descriptions of the options in sync with the actual options). That's why I adjusted the tooltip you're asking about to match this style.

Reply:
But the difference is that the word is highlighted in this case which gives it a reason to be capitalized. In the other translation its not and has no reason to be capitalized.

Reply:
What would you like me to rename "Push to Talk" to? This may be the only instance where this term is not capitalized after the change, but that may not necessarily be a bad thing.

Reply:
Yeah I'd leave it lowercase in that case

Reply:
If you say so, then so be it.